### PR TITLE
travisci to github actions

### PR DESCRIPTION
**Pull request by waka**
Stop Travis CI and move to GitHub Actions

### CODE REVIEW

File: seed-snapshot.gemspec [hunk 1]
```diff
@@ -17,11 +17,5 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
 
-  spec.add_development_dependency 'bundler'
-  spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'pry'
-  spec.add_development_dependency 'minitest'
-  spec.add_development_dependency 'erubis'
-  spec.add_development_dependency 'mysql2', '>= 0.4.4', '< 0.6.0'
```
Comment (keik) [inline seed-snapshot.gemspec:25]: I see https://github.com/rubygems/bundler/pull/7222

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: 'activerecord_unittest'
```
Comment (keik) [inline .github/workflows/main.yml:24]: This database is required implicitly to create in local environment, and this is a little bit implicit.
I think that It is better to write doc or make script to do this.

**Review by keik — APPROVED** [review]
LGTM!
nice~~

**Comment by waka** [comment]
arigato!

**Pull request merged**